### ISSUE TITLE
Add travis tests

### ISSUE THREAD

**Issue by JJ** [jmv74211/TFM_security_system_PI#32]
Functional, unit tests don't depend on any particular implementation, and should be tested anyway.

**Comment by jmv74211** [comment]
Yes, so far I run the tests manually before committing. It would be nice to use Travis's tool to run the tests automatically, but for reasons of time I haven't been able to implement it for now.
